src/Searchlight/FetchRequest.cs

```diff
@@ -26,6 +26,11 @@ public class FetchRequest
         /// For pagination: The page size to use for this query.
         /// </summary>
         public int? pageSize { get; set; }
+        
+        /// <summary>
+        /// For pagination: Sets the maximum page size for this query.
+        /// </summary>
+        public int maximumPageSize { get; set; }
```

Comment:
This is incorrect.  Maximum page size should be set in the SearchlightEngine, not in the request.